lib/traits.ts

```diff
@@ -43,6 +43,327 @@ export type TraitsArray = [
   string,
 ];
 
+export type ItemLayer = {
+  name: string;
+  description: string;
+  thumbnail: string;
+  layer: string;
+  type: EquippableTraitType;
+};
+
+export const DEFAULT_ITEMS: Array<ItemLayer> = [
```

Comment:
Everything is working perfectly when creating the item, but I think these default item names are actually breaking the ability to visually equip them. Looking into it--will try to push a fix ASAP

Reply:
Luckily was an easy fix: 4aa3cc88fca6abc91edd456bf65e30e265a7777d